templates/googleapps/index.tpl

```diff
@@ -34,11 +34,12 @@
 <p>Tu peux créer ce compte indépendant <em>Google Apps pour Polytechnique.org</em> en utilisant le
   formulaire ci-dessous.
 </p>
+{/if}
+
 <p><strong>Attention&nbsp;:</strong> ce compte Google Apps est hébergé par <em>Google</em>, et non
   par Polytechnique.org. Nous te déconseillons d'y conserver des informations
   confidentielles ou sensibles&nbsp;!<br />&nbsp;
 </p>
-{/if}
```

Comment:
This moves the `{/if}` outside of the `{if !$has_password_sync}`, which is good.
Still this is inside `{if !$account->g_status}`, suggesting is in a part shown only before the account is created. This information feels relevant to be shown irrespective of the account status.